src/Core/Models/Api/Request/SsoConfigRequestModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Bit.Core.Models.Api
+{
+    public class SsoConfigRequestModel
+    {
+        public long Id { get; set; }
```

Comment:
So here's the crux of numeric `IDENTITY` seed values in SQL and C# models... this value should be `long?` (nullable) 'cause the code is not assigning the value, the database is, so when Dabber maps it, it'll need to be `null`, otherwise you'll get a `0L` (default value for long) and that'll go ka-boom 'cause SQL will attempt to insert it, or at the least evil path, Dabber will not map the identity value back out appropriately.

Reply:
@cscharf I don't think SQL will try to insert this since we don't set it's value to be inserted in the sprocs.

Reply:
Correct, but EF will when we implement other DB platforms (like a postgres sequence, an autonumber, etc.)